dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -391,6 +391,14 @@ protected void githubWebhookRelease(String repository, String username, String g
         }
     }
 
+    private String createMessageFromException(Exception ex) {
+        String message = ex instanceof CustomWebApplicationException ? ((CustomWebApplicationException)ex).getErrorMessage() : ex.getMessage();
+        if (ex instanceof DockstoreYamlHelper.DockstoreYamlException) {
+            message = "Error reading .dockstore.yml: " + message;
+        }
```

Comment:
DockstoreYamlHelper already has a constant for `Error reading .dockstore.yml`, so could re-use that here